### DISCUSSION TITLE
RemoteApp goes under the same icon tray

### DISCUSSION THREAD

**Discussion by hbatra917** [FreeRDP/FreeRDP#8514] [General]
I just had a little concern regarding the fact that when using FreeRDP to start numerous applications, they all appear in the same app tray with the same programme icon.
As an illustration, if I first open Excel and then Powerpoint, Powerpoint appears in the Excel icon tray.

One more thing: when I use FreeRDP for a remote app, the app name does not work and displays as RAIL:00180148. 
![Screenshot from 2022-12-09 17-07-35](https://user-images.githubusercontent.com/120174363/206694205-55814fdb-cd6c-4589-a630-5633b8ab82f6.png)

**Comment by akallabeth** [comment]
This is due to the remote applications not being applications from the local desktop point of view. (you need `.desktop` files for most of the menus to pick them up) but just windows of an application (e.g. `xfreerdp`)
As for the name, the remote desktop creates a lot of (unnamed) windows and whichever is the first will be displayed as the name.

So, while this is annoying we currently do not have an easy fix, but if someone has a good idea we are happy to implement it.

**Reply by hbatra917** [reply]
Thankyou for that.

**Comment by hbatra917** [comment]
I also want to ask about the camera redirection in the FreeRDP.
I wanted to redirect camera but the camera is not showing on the Remote Desktop.
Please Suggest me way out.
